Source/player.h

```diff
@@ -402,6 +402,27 @@ struct PlayerStruct {
 	 * @brief Resets all Data of the current PlayerStruct
 	 */
 	void Reset();
+
+	/**
+	 * @brief Calculates the players current Hit Points as a percentage of their max HP and stores it for later reference
+	 *
+	 * The stored value is unused...
+	 * @see _pHPPer

```

Comment:
Is the value used by vanilla? If not then let's just skip it.

Reply:
I'm guessing the original intent was to refer to this value when drawing the UI and only recalculate it when the players max or current HP changes. I'd like to go back to that behaviour eventually since it doesn't really need to be recalculated every frame.